lib/javascript/jplayer.playlist.ext.js

```diff
@@ -12,8 +12,8 @@ jPlayerPlaylist.prototype._createListItem = function(media) {
         }
         attrClass += "\" ";
 
-        var listItem = "<li" + attrClass + " name=\"" + $(jplaylist.cssSelector.playlist + " ul li").length + "\" " +
-                "data-poster=\"" + media.poster + "\" data-media_id=\"" + media.media_id + "\" data-album_id=\"" + media.album_id + "\" data-artist_id=\"" + media.artist_id + "\" " +
+        var listItem = "<li" + attrClass +
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals would make this readable. But I don't know if Ampache cares about Internet Explorer support.

Reply:
I'm not sure about this, would be nice to know if anyone still uses it.